docs/application/dotnet/guides/nui/text.md

```diff
@@ -150,6 +150,16 @@ To align the text in a text label:
 
 
 
+### UTF16 or UTF32 character encoding
+
+The text property can accept UTF16 or UTF32 character encoding

```

Comment:
@surbhig12 I think checking the colons before the code block is missed.

```suggestion
The text property can accept UTF16 or UTF32 character encoding:

```

Reply:
THank you for pointign this out.